src/LtiLineitem.php

```diff
@@ -143,4 +146,16 @@ public function setEndDateTime($value)
 
         return $this;
     }
+
+    public function getGradesReleased()
+    {
+        return $this->grades_released;
+    }
+
+    public function setGradesReleased($value)
```

Comment:
Is it possible to add typing to this variable? I looked at the spec, and it doesn't offer any indication as to what type(s) are allowed here. Is this a boolean, string, timestamp, black hole, something else?

Edit: I found some examples in the spec doc, and it looks like they're using bools.

```suggestion
    public function setGradesReleased(bool $value)
```